cpp/tests/utilities/column_utilities.cu

```diff
@@ -371,31 +371,34 @@ struct column_property_comparator {
   }
 };
 
+template <typename DeviceComparator>
 class corresponding_rows_unequal {
  public:
-  corresponding_rows_unequal(table_device_view d_lhs,
-                             table_device_view d_rhs,
+  corresponding_rows_unequal(table_device_view /*d_lhs*/,
+                             table_device_view /*d_rhs*/,
```

Comment:
The parameter `d_lhs` and `d_rhs` are used only here: https://github.com/rapidsai/cudf/pull/12777/files#diff-4292feecdf2f0106ee3b94779dd191033fd27c494bdffb944685c94fade6fc74R425.
This check can also be moved to the caller site. So we can remove these two parameters entirely.

Reply:
You are right, in fact the check is not even needed because the caller site only works with 1 column anyway